Rakefile

```diff
@@ -5,3 +5,28 @@ task :default do
 end
 
 require "logstash/devutils/rake"
+
+desc "Compile and put filewatch jar into lib/jars"
+task :vendor do
+  exit(1) unless system './gradlew clean jar'
+  puts "-------------------> built filewatch jar via rake"
+end
+
+desc "Run full check with custom Logstash path"
+task :custom_ls_check, :ls_dir do |task, args|
```

Comment:
not sure why this is required? what I usually do is
```
LOGSTASH_SOURCE=1 bundle exec ...
```
not sure we need/want to add yet another custom rake task?

Reply:
I use this to run the specs against any downloaded LS as well as my LS dev repo.
I have this:
```
logstash-2.1.3
logstash-2.2.2
logstash-2.3.4
logstash-2.4.0
logstash-2.4.1
logstash-5.0.0
logstash-5.2.1
logstash-5.2.2
logstash-5.3.0
logstash-5.4.3
logstash-5.5.0
logstash-5.5.1
logstash-5.6.2
logstash-5.6.4
logstash-5.6.8-SNAPSHOT
logstash-6.0.0
logstash-6.1.0
logstash-6.1.1
logstash-6.1.2
logstash-6.2.1
logstash-6.2.2
```

Reply:
ok, that looks super useful, my problem is that this is specific to your workflow and is not really documented or used anywhere else, or is it?